boot/storage.txt

```diff
@@ -0,0 +1,485 @@
+    strictly by the motherboard used in a given system. The latter, however,
+    refer to particular partition types: GUID Partition Tables versus the
+    Master Boot Record. These types directly limit the structures available
+    on any given disk drive - both the number of partitions available, as 
+    well as the size of any given partition.
+    
+    An important point of consideration is whether or not a separate HOME 
+    partition should be used. This is useful in cases where users wish to keep
+    there data in a location separate from their operating system, for instance
+    in cases of critical failures requiring the OS to be reinstalled, or if 
+    users wish to encrypt their personal data separately from their core OS.
+    This separation between / and /home allow users to easily migrate their data
+    across many different operating systems; they need only remount the HOME
+    partition to /home on the new system. In general, this is recommended.
+
+    In addition to a separate HOME, several other additional partition options
+    exist. For instance, a separate var (/var) and data partitions can be used.
+    Separate data partitions are useful in instances where users would like to 
+    easily share data across different operating systems, as allowing write 
+    access to a user's home directory is a security risk. A separate /var is 
+    useful in niche cases where /usr is read-only. For more detailed information
+    on filesystem structures, refer to the Filesystem Hierarcy Standard [0].
+    Additionally, the Linux From Scratch project has some useful rationales [1].
+
+
+    [1.2] Tools
+    ____________________________________________________________________________
+
+    Depending on which live environment the user chooses for installing a KISS
+    Linux system, the tools available vary. On most UNIX systems, the following
+    can usually be expected:
+
+    - fdisk: a dialogue driven partition manipulator [2]
+    - gdisk: GPT fdisk (note that fdisk also supports GPT) [3]
+    - cfdisk: a partition editor with a curses-based UI, part of util-linux [4]
+    - parted: a GNU partition editor [5]
+
+    It is also possible to use the live CD version of parted, GParted [6], to 
+    partition disks, and it offers many other useful features for disk 
+    management and data recovery.
+
+    In general, most programs can handle the majority of use-cases. The tool of 
+    choice is almost entirely user preference. However, some utilities have 
+    powerful and advanced features that can render data on disks damaged. fdisk
+    is a one such program, and is useful in cases where more fine-grained
+    control is required. For general system setup, however, any tool will do.
+    
+    
+[2.0] Partitioning Schemata
+________________________________________________________________________________
+
+Because disk setup is in general difficult to modify after a particular
```

Comment:
```
Great care should be taken when picking a partition structure as changing it afterwards is (generally) a difficult endeavor. Ensure that the chosen structure meets your use case. ...
```

Just a suggestion/idea (I don't like sentences starting with "because"). Feel free to reword this as you wish.

Reply:
'Because' is a dirty word and I should know better.